### PR TITLE
WIP: Added some initial base styles, comments and tweaks

### PR DESCRIPTION
**Pull request by NickColley**
Adds:
- HTML Validation
- Basic responsive styles
- Clear tracker function
- Started adding some comments

I think maybe my comments are a bit rambling sometimes maybe? Let me know what you think.

Larger viewports:
image

Smaller viewports:
image

### CODE REVIEW

File: assets/app.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
+/* Core overides */
```
Comment (NickColley) [inline assets/app.css:1]: Could do with commenting the CSS properly, explaining these sections.

**Comment by gr2m** [comment]
Love it! I’d say let’s merge it in and go from there?

File: assets/app.js [hunk 2]
```diff
@@ -1,11 +1,36 @@
 /* global hoodie */
+/**
+ * Hoodie is global to the project as it's loaded in the page before hand.
+ * Try executing 'window.hoodie' and 'hoodie' in your browser dev tools console.
+ * Look up Scope in JavaScript for more information
+ * http://www.smashingmagazine.com/2009/08/what-you-need-to-know-about-javascript-scope/
+ * Generally you want to avoid global variables in your applications as it will make your application unpredicatable.
+ */
```
Comment (gr2m) [inline assets/app.js]: How about we remove this entire thing, including `/* global hoodie */`, this is not important for new developers, and experienced developers will already know. I wouldn't like for new developers to be this the first thing they see when they look into app.js

File: index.html [hunk 3]
```diff
@@ -2,40 +2,66 @@
...
+        <div class="grid-col">
+          <form class="js-tracker-input">
+            <label class="form-label" for="amount">Amount</label>
+            <input
+              class="form-input"
+              type="number"
+              name="amount"
+              id="amount"
+              required
+              min="1"
```
Comment (gr2m) [inline index.html]: Let’s remove `min`, people might want to track negative numbers, too

**Comment by gr2m** [comment]
Also you can add a preview to your PR like this:
https://rawgit.com/nickcolley/my-first-hoodie/something-tracker/index.html

File: assets/app.js (hunk 2 above)
Comment (NickColley) [inline assets/app.js]: :+1:

File: index.html (hunk 3 above)
Comment (NickColley) [inline index.html]: :+1:

**Comment by NickColley** [comment]
Ah that's really neat!

**Comment by varjmes** [comment]
Niiiiiiiiiiiiiiiiiiiiiiiiiiiiiice! I love it.

**Comment by boennemann** [comment]
Could you please make these PRs against the next branch, because that's what is used by hoodie@next without affecting current hoodie apps. Thank you :)

**Pull request merged**